lib/ulid/generator.rb

```diff
@@ -27,7 +29,7 @@ def generate_bytes(time = Time.now)
     private
 
     def encode(n, length)
-      e = '0' * length
+      e = Array.new(length, 48) # '0'.ord
```

Comment:
Can we pass `'0'.ord` as the 2nd argument instead of the hardcoded 48?
Alternatively, that could also be `ENCODING.first` or even be extracted to its own constant.

Reply:
@rafaelsales Sure, done.